.pre-commit-config.yaml

```diff
@@ -47,7 +47,7 @@ repos:
         args: ["--verbose"]
     # Analyze the code style and report code that doesn't adhere.
   - repo: https://github.com/psf/black
-    rev: 23.7.0
+    rev: 24.10.0
```

Comment:
Same for this - is there something that's changing this in the VS code or something like that?